ecs-agent/tmds/handlers/v4/handlers_test.go

```diff
@@ -24,21 +24,23 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
nit(non-blocking): ordering of imports is a bit inconsistent with our existing format. (1. Built-in packages, 2. Local packages, 3. External packages)

Reply:
sounds good. will update this.

Reply:
Updated.